src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Network</PackageId>
 		<Description>Provides management capabilities for Network services.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>17.0.0-preview</Version>
+		<Version>18.0.0-preview</Version>
 		<PackageTags>Microsoft Azure Network management;Network;Network management;</PackageTags>    
 		<PackageReleaseNotes>
 		<![CDATA[
```

Comment:
Please update the package release notes for this version

Reply:
Fixed